packages/styles/scss/patterns/blocks/leadspace-block/_leadspace-block.scss

```diff
@@ -15,19 +15,11 @@
     &__title {
       width: 90%;
       color: $text-01;
-      padding-top: $layout-05;
+      padding-top: $layout-03;
       @include carbon--type-style('expressive-heading-06', true);
 
-      @include carbon--breakpoint('lg') {
-        padding-top: $layout-05;
-      }
-
       @include carbon--breakpoint('md') {
-        padding-top: $layout-03;
-      }
```

Comment:
@ariellalgilmore Looks like this was correct, but the order needed to be changed (`md` first, then `lg`). The way you have it now, it's `$layout-05` for both md/lg breakpoints.